pkg/resources/digitalocean/cloud.go

```diff
@@ -62,6 +64,13 @@ type Cloud struct {
 	RegionName string
 }
 
+type DOInstanceGroup struct {
+	ClusterName       string
+	InstanceGroupName string
+	GroupType         string   // will be either "master" or "worker"
```

Comment:
Use `pkg/apis/kops/InstanceGroupRole` as the type?

Reply:
I'll keep a note to update this when I touch the rolling update feature. I'll add a new tag when I create a droplet that holds this information, so I can extract it from the droplet tag.
Thanks for your help in reviewing this @johngmyers !

Reply:
You can get the group role from the instancegroup spec of the ig that was in the tag.